frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/FossologyAwarePortlet.java

```diff
@@ -174,4 +183,28 @@ protected void putReleasesAndProjectIntoRequest(PortletRequest request, String p
         request.setAttribute(PortalConstants.RELEASES_AND_PROJECTS, releaseClearingStatuses);
     }
 
+    protected void serveFossologyReloadReport(ResourceRequest request, ResourceResponse response) {
+        Iface componentClient = thriftClients.makeComponentClient();
+        FossologyService.Iface fossologyClient = thriftClients.makeFossologyClient();
+        String releaseId = request.getParameter(RELEASE_ID);
+        JSONObject jsonObject = JSONFactoryUtil.createJSONObject();
+        try {
+            Release release = componentClient.getReleaseById(releaseId, UserCacheHolder.getUserFromRequest(request));
+            Set<ExternalToolProcess> fossologyProcesses = SW360Utils.getNotOutdatedExternalToolProcessesForTool(release,
+                    ExternalTool.FOSSOLOGY);
+            fossologyClient.triggerReportGenerationFossology(releaseId, UserCacheHolder.getUserFromRequest(request));
```

Comment:
Please check if we can use `RequestStatus` returned from `triggerReportGenerationFossology()` , to validate  Success and Failure.